### PR TITLE
Added privacy page

### PR DESCRIPTION
**Pull request by milindk8**
#Description 

Added privacy page & links in all HTML pages

### CODE REVIEW

File: public/privacy.html [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+          </div>
+        <div>Made with <span class="text-span">♥</span> in India</div>
+      </div>
+    </div>
+  </div>
+  <script src="https://d3e54v103j8qbb.cloudfront.net/js/jquery-3.5.1.min.dc5e7f18c8.js?site=61aeec2bcab0f63538b9256e" type="text/javascript" integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0=" crossorigin="anonymous"></script>
+  <script src="js/webflow.js" type="text/javascript"></script>
+  <!-- [if lte IE 9]><script src="https://cdnjs.cloudflare.com/ajax/libs/placeholders/3.0.2/placeholders.min.js"></script><![endif] -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/1.7.1/clipboard.min.js"></script>
+  <script>
```
Comment (vivek-devtron) [inline public/privacy.html:356]: We can remove this unnecessary script

File: public/privacy.html [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+ new Clipboard('.btn');
+</script>
+  <!--  F’in sweet Webflow Hacks  -->
+  <script>
+// when the DOM is ready
+$(document).ready(function() {
```
Comment (vivek-devtron) [inline public/privacy.html:362]: We can remove this unnecessary script

File: public/privacy.html (hunk 1 above)
Comment (milindk8) [reply public/privacy.html:356]: This script is necessary for hamburger menu to work

File: public/privacy.html (hunk 2 above)
Comment (milindk8) [reply public/privacy.html:362]: Removed

**Review by vivek-devtron — APPROVED** [review]

**Review by sohel-rp — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**